build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
   id 'checkstyle'
   id 'io.spring.dependency-management' version '1.0.11.RELEASE'
   id 'jacoco'
-  id 'org.springframework.boot' version '2.5.5'
+  id 'org.springframework.boot' version '2.5.7'
```

Comment:
https://tools.hmcts.net/jira/secure/RapidBoard.jspa?rapidView=1608&projectKey=PUB&view=detail&selectedIssue=PUB-1005#:~:text=To%20upgrade%20code%20repositories%20to%20use%202.6.2

please see ticket for upgrading

Reply:
It was updated in master branch but not in pre-release. Confirmed it with @ChrisS1512.